### PR TITLE
adds an optional accountId for renderer topic subscription rights

### PR DESCRIPTION
**Pull request by mbonig**
I want to be able to subscribe to the sns topic for when the renderer has completed. This change allows for an optional externalAccountSubscribers to be passed to the CatalogStack and creates a topic policy accordingly.

### CODE REVIEW

File: packages/catalog/config.ts [hunk 1]
```diff
@@ -24,5 +24,8 @@ export const prod: CatalogStackProps = {
   env: {
     account: '499430655523',
     region: 'us-east-1',
-  }
-};
\ No newline at end of file
+  },
+  externalAccountSubscribers: [
+    "536309290949"  // matthew.bonig@gmail.com
```
Comment (eladb) [inline packages/catalog/config.ts:29]: Do you mind adding the purpose of the subscription here?

Comment (mbonig) [reply packages/catalog/config.ts]: I added the reason I want the subscription.

**Comment by mbonig** [comment]
bump

File: packages/catalog/config.ts [hunk 2]
```diff
@@ -24,5 +24,8 @@ export const prod: CatalogStackProps = {
   env: {
     account: '499430655523',
     region: 'us-east-1',
-  }
-};
\ No newline at end of file
+  },
+  externalAccountSubscribers: [
+    "536309290949"  // so matthew.bonig@gmail.com can auto-build his constructs when new versions of `core` are released.
```
Comment (eladb) [inline packages/catalog/config.ts:29]: Why not use something like dependabot?

Comment (mbonig) [reply packages/catalog/config.ts:29]: I think it'd be fine for the updating of my constructs, but I had some other ideas in mind outside of just those github repos.

Comment (eladb) [reply packages/catalog/config.ts:29]: ok, no problem

**Review by eladb — APPROVED** [review]

**Pull request merged**